notebooks/book2/03/hierarchical_binom_rats.ipynb

```diff
@@ -0,0 +1,1831 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.            7,
</pre><p>Can you reformat this data vector so it takes less space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Unfortunately, there was no way given that how black format list. So I have moved data to `probml_data`.

Reply:
Yes I suspect this was a black problem. Is there no way to tell black to ignore formatting rules for a given line?
This comes up in several notebooks where we hard-code a small dataset. It seems a bit of a pain to have to store the data in probml_data.